MetadataExtractor/Formats/Exif/ExifReader.cs

```diff
@@ -17,7 +17,7 @@ public sealed class ExifReader : JpegSegmentWithPreambleMetadataReader
 
         private static ReadOnlySpan<byte> ExifPreamble => "Exif\x0\x0"u8;
 
-        public static bool StartsWithJpegExifPreamble(byte[] bytes) => bytes.AsSpan().StartsWith(ExifPreamble);
+        public static bool StartsWithJpegExifPreamble(byte[] bytes) => bytes.AsSpan.StartsWith(ExifPreamble);
```

Comment:
Should AsSpan -> Span?

Reply:
UGH! Dumb mistake! And good catch. Fixed.